src/extract-cli.spec.js

```diff
@@ -0,0 +1,13 @@
+const fixtures = require('./test-fixtures.js');
+const fs = require('fs');
+const util = require('util');
+
+const exec = util.promisify(require('child_process').exec);
+
+describe('Command Line Usage', () => {
+  it('should output a correct POT file for vue component with $gettext used in template and data', async() => {
+    fs.writeFileSync('/tmp/TestComponent1.vue', fixtures.VUE_COMPONENT_WITH_GETTEXT_IN_TEXT_AND_DATA);
+    const {stdout} = await exec('node ./src/extract-cli.js --attribute v-translate /tmp/TestComponent1.vue ');
+    expect(stdout).toEqual(fixtures.CLI_OUTPUT_VUE_COMPONENT_WITH_GETTEXT_IN_TEXT_AND_DATA);
+  });
+});
```

Comment:
yes ! you introduced the first CLI test, which I was way too lazy to introduce back then :p since it required running a process and may be difficult to maintain. But I'm all for it :)

Pay attention, though, that:
- Jest runs the tests in parallel by default.
- There should NOT be another test accessing your "test fixture file" in the same time, it'll become flaky
- this test is not platform independent
- usually it's best to try and find a way to pipe the data from memory, instead of leaving temporary files behind; or use a library that does it for you.

But again, at least it works and adds significant value to the code since... Well it was broken and untested ^^

Maybe you can put my comments there so that the next contributors would know what to do or not do.

Reply:
Added comments and example to have one tmp-file per test